mpu6050/mpu6050.c

```diff
@@ -193,7 +250,91 @@ static void class_remove_group(struct class *class, struct class_attribute **att
 		class_remove_file(class, *attrs);
 }
 
-static struct class *mpu6050_class;
+/*
+ * time: gyro=gyroX:gyroY:gyroZ acc=accX:accY:accY
+ */
+static void mpu6050_seq_print_one(struct seq_file *seq, struct mpu6050_data *data)
+{
+	seq_printf(seq, "time: %lu gyro:%hd:%hd:%hd acc=%hd:%hd:%hd\n",
+			data->timestamp,
+			data->gyro_values[X],
+			data->gyro_values[Y],
+			data->gyro_values[Z],
+			data->accel_values[X],
+			data->accel_values[Y],
+			data->accel_values[Z]);
+}
+
+static int mpu6050_current_seq_show(struct seq_file *seq, void *d)
+{
+	int idx = (int)seq->private;
+	struct mpu6050_data *data = &mpu6050_data_samples[idx];
+
+	mpu6050_seq_print_one(seq, data);
+	return 0;
+}
+
+static int mpu6050_all_seq_show(struct seq_file *seq, void *d)
+{
+	int i;
+
+	for (i = 0; i < SAMPLES_COUNT; i++) {
```

Comment:
If you have already got more than `SAMPLES_COUNT` readings,
you'll output values in wrong order.
In such case you should start from `mpu6050_current_sample_idx + 1` and make a round.